docs/source/conf.py

```diff
@@ -201,7 +201,12 @@
 
 # General information about the project.
 project = u'Apache Arrow'
-copyright = f'2016-{datetime.datetime.now().year} Apache Software Foundation'
+copyright = (
+    f"2016-{datetime.datetime.now().year} Apache Software Foundation.\n"
```

Comment:
`\n` doesn't add a newline. We could try multiline copyright text with comma separated items: https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-copyright like:

```
copyright = (
    f"2016-{datetime.datetime.now().year} Apache Software Foundation.",
    "Apache Arrow, Arrow, Apache, the Apache feather logo, and the Apache Arrow project logo are either registered",
    "trademarks or trademarks of The Apache Software Foundation in the United States and other countries"
)
```
but would need to use newer version of Sphinx ([7.0.1](https://www.sphinx-doc.org/en/master/changes.html#release-7-1-0-released-jul-24-2023)). I think we should unpin Sphinx anyways.

Breaking the copyright text into multiple lines would also make it smaller in width and so the text on the right (PyData SphinxTheme text) could be kept all in one line. See http://crossbow.voltrondata.com/pr_docs/39505/

Reply:
Good point .. the `\n` indeed doesn't do anything. 
Updating sphinx is indeed something we should do, but from looking at that PR, the multiline copyright might not be exactly what we want here, because from the test added in that PR, it seems that it adds the `© Copyright ` to each line then, which we don't want (https://github.com/sphinx-doc/sphinx/pull/10983/files#diff-da24b355349b53e4c8047034767d008e4c2ce9ea6c8933a3acd50d0828f00528)

Reply:
It's a bit annoying that the layout is not looking good with this long copyright text, but I am afraid that this is only something that can be fixed in the theme (or we could override the footer block in our layout). 

Also on mobile it doesn't look good, but that's again an issue with the upstream theme as well.

Reply:
Oh you are correct. Having `© Copyright` on each line sure isn't what we want.

Reply:
Yeah, there is not much we can do about it really.